mlir/test/Examples/standalone/test.wheel.toy

```diff
@@ -0,0 +1,28 @@
+# There's no real issue with windows here, it's just that some CMake generated paths for targets end up being longer
+# than 255 chars when combined with the fact that pip wants to install into a tmp directory buried under
+# C/Users/ContainerAdministrator/AppData/Local/Temp.
+# UNSUPPORTED: target={{.*(windows).*}}
+
+# RUN: export CMAKE_BUILD_TYPE=%cmake_build_type
+# RUN: export CMAKE_CXX_COMPILER=%host_cxx
+# RUN: export CMAKE_CXX_COMPILER_LAUNCHER=%hostcxx_compiler_launcher
+# RUN: export CMAKE_C_COMPILER=%host_cc
+# RUN: export CMAKE_C_COMPILER_LAUNCHER=%hostc_compiler_launcher
+# RUN: export CMAKE_GENERATOR=%cmake_generator
+# RUN: export LLVM_USE_LINKER=%llvm_use_linker
+# RUN: export MLIR_DIR="%mlir_cmake_dir"
+
+# RUN: %python -m pip uninstall -y standalone-python-bindings
```

Comment:
Should we be doing this in an environment? (e.g., could this have non-test local effects?)

Reply:
I tried but the GH runners don't have `python3-venv` installed so it's not possible. Whether there could be side-effects (no pun intended) I think it's unlikely that someone at some point actually has a useful package called `standalone-python-bindings` but I could name mangle it more.

Reply:
all cleaned up